[2023-02-02T05:17:45.030+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v4.third_task scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2023-02-02T05:17:45.377+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v4.third_task scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2023-02-02T05:17:45.401+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T05:17:45.416+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-02-02T05:17:45.420+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T05:17:45.892+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): third_task> on 2022-01-30 00:00:00+00:00
[2023-02-02T05:17:46.198+0000] {standard_task_runner.py:55} INFO - Started process 4631 to run task
[2023-02-02T05:17:46.357+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'third_task', 'scheduled__2022-01-30T00:00:00+00:00', '--job-id', '596', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpqy20i5aa']
[2023-02-02T05:17:46.416+0000] {standard_task_runner.py:83} INFO - Job 596: Subtask third_task
[2023-02-02T05:17:47.776+0000] {task_command.py:388} INFO - Running <TaskInstance: our_first_dag_v4.third_task scheduled__2022-01-30T00:00:00+00:00 [running]> on host 178c757a8fee
[2023-02-02T05:17:48.693+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=archana
AIRFLOW_CTX_DAG_ID=our_first_dag_v4
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-30T00:00:00+00:00
[2023-02-02T05:17:48.757+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-02T05:17:48.804+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task3 and will be running after task1 as task2']
[2023-02-02T05:17:49.247+0000] {subprocess.py:86} INFO - Output:
[2023-02-02T05:17:49.305+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task1 as task2
[2023-02-02T05:17:49.312+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-02-02T05:17:49.575+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=third_task, execution_date=20220130T000000, start_date=20230202T051745, end_date=20230202T051749
[2023-02-02T05:17:50.001+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-02T05:17:50.241+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
